evm/src/fuzz/mod.rs

```diff
@@ -100,9 +99,10 @@ impl<'a> FuzzedExecutor<'a> {
             let call = self
                 .executor
                 .call_raw(self.sender, address, calldata.0.clone(), 0.into())
-                .expect("Could not call contract with fuzzed input.");
+                .map_err(|_| FuzzError::FailedContractCall)
+                .unwrap();
             let state_changeset =
-                call.state_changeset.as_ref().expect("We should have a state changeset.");
+                call.state_changeset.as_ref().ok_or(FuzzError::EmptyChangeset).unwrap();
```

Comment:
we could change the return value to `FuzzResult` and not panic here, instead bubble up the error

Reply:
removed all the panic-able code and bubble up the errors